espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/MobileBackdoorMethod.java

```diff
@@ -0,0 +1,43 @@
+package io.appium.espressoserver.lib.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+@SuppressWarnings("unused")
+public class MobileBackdoorMethod {
+    String name;
+
+    @Nullable
+    List<Object> args;
+
+    public String getName() {
+        return name;
+    }
+
+    @Nullable
+    public List<Object> getArgs() {
+        if (args == null) {
+            return null;
+        }
+
+        List<Object> list = new ArrayList<>();
+
+        for (Object object : args) {
+            if (object instanceof Double) {
+                Double d = (Double) object;
+                if ((d % 1) == 0) {
+                    list.add(d.intValue());
```

Comment:
I would say it makes sense to explicitly distinguish between long and int values

Reply:
I am not sure how to handle this because gson is parsing integers as float. https://stackoverflow.com/questions/15507997/how-to-prevent-gson-from-expressing-integers-as-floats/15508288
I might need help here as I am not too sure what to do

Reply:
We should check if the resulting value is greater than MAX_INT and then assign longValue to the result instead of intValue. Also it would be handy to explain that in comments

Reply:
Added a check